website/content/docs/commands/alloc/stop.mdx

```diff
@@ -18,8 +18,10 @@ nomad alloc stop [options] <allocation>
 
 The `alloc stop` command requires a single argument, specifying the alloc ID or
 prefix to stop. If there is an exact match based on the provided alloc ID or
-prefix, then the alloc will be stopped and rescheduled. Otherwise, a list of
-matching allocs and information will be displayed.
+prefix, then the alloc will be stopped, otherwise, a list of
+matching allocs and information will be displayed. If the stopped allocation
+belongs to a service or batch job they will rescheduled according to their 
+reschedule policy, [system allocs will not] and will show up as completed.
```

Comment:
```suggestion
reschedule policy. [System allocs will not][] and will show up as completed.
```

Reply:
neat, I didn't realize that the footer `[link content]`s were case-insensitive like this, but looks like they are: https://nomad-c0iq06nwa-hashicorp.vercel.app/nomad/docs/commands/alloc/stop#usage